### PR TITLE
:recycle: some changes

### PR DESCRIPTION
**Pull request by AsliddinBuriev**

### CODE REVIEW

File: lib/test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+hello  world
```
Comment (ilyosbekkk) [inline lib/test.md:1]: your code is awesome

**Review by ilyosbekkk — APPROVED** [review]
confirmed

File: lib/test.md (hunk 1 above)
Comment (AsliddinBuriev) [reply lib/test.md:1]: thanks man!

Comment (ilyosbekkk) [reply lib/test.md:1]: approved

**Pull request merged**